### DISCUSSION TITLE
Translate code

### DISCUSSION THREAD

**Discussion by Dontfwihme** [Flinesoft/BartyCrouch#278] [General]
Hello, 
This app is crazy awesome. I want to learn so much how to figure all this shit out but it's like trying to learn a different language anyway in my process of working on my project. I'm trying to figure out how to get access to data on a device aka apple phone with the serial number and the IMEI and I stumbled across this stuff I'm getting closer and I don't know how to read it so if someone can help that be awesome and teach me where to go to lean and read more.

**Comment by Jeehut** [comment]
Hi, I'm not sure if reading data from a phone is the right use case for BartyCrouch. This is a tool to help developers improve their translation workflow and it's all described in detail in the README and the article linked within.

If you have a specific question about BartyCrouch, please ask it. But this is not a support forum for any kind of programming question, let alone a forum to help anyone read data from phones which sounds like an illegal action to me (but maybe I misunderstood what you were trying to achieve).

In any case, StackOverflow or other Q&A website is the better place to ask your question.
